libs/platform/src/lib/components/list/standard-list-item/standard-list-item.component.html

```diff
@@ -0,0 +1,110 @@
+<!--  list item navigation with Arrow-->
+<ng-container *ngIf="navigationIndicator || navigated && !(noDataText!==null && noDataText!== undefined) ">
+    <li #listItem fd-list-item [attr.id]="id" [selected]="selected" class="fd-list__item--link" tabIndex="0"
+        [attr.aria-selected]="selected" (keydown)="onKeyDown($event)" (keyup)="onKeyUp($event)"
+        [class.fd-list-item__unread]="unRead">
+        <a #link class="fd-list__link fd-link fdp-link-truncate__txt" [routerLink]="routerLink"
+            [class.fd-list__link--navigation-indicator]="navigationIndicator">
+            <ng-container *ngTemplateOutlet="hasByLine ? byLineTemplate: commonTemplate"></ng-container>
+        </a>
+    </li>
+</ng-container>
+
+<!--  list item without navigation and used for simple list display-->
+<ng-container *ngIf="!navigationIndicator && !navigated && !(noDataText!==null && noDataText!== undefined)">
+    <li #listItem fd-list-item [attr.id]="id" [selected]="selected" (keydown)="onKeyDown($event)" tabIndex="0"
+        (keyup)="onKeyUp($event)" [class.fd-list-item__unread]="unRead">
+        <ng-container *ngTemplateOutlet="hasByLine ? byLineTemplate: commonTemplate"></ng-container>
+    </li>
+</ng-container>
+
+<ng-container *ngIf="noDataText!==null && noDataText!== undefined">
+    <li #listItem fd-list-item [attr.id]="id" tabindex="0" class="fd-list__item--no-data">
+        <span fd-list-title [attr.aria-label]="noDataText" [attr.title]="noDataText">{{noDataText}}</span>
+    </li>
+</ng-container>
+
+<ng-template #commonTemplate>
+    <ng-container *ngTemplateOutlet="selectionTemplate"></ng-container>
+    <span *ngIf="title" fd-list-title [attr.aria-label]="title" [attr.nowrap]="titleWrap"
+        [attr.title]="title">{{title}}</span>
+    <span fd-list-secondary [attr.aria-label]="secondary" *ngIf="secondary!== null && secondary!== undefined"
+        [attr.title]="secondary" [attr.nowrap]="secondaryWrap">
+        {{secondary}}
+    </span>
+    <span fd-list-secondary class="fd-list__counter" [attr.aria-label]="counter"
+        *ngIf="counter!== null && counter!== undefined" [attr.title]="counter">
+        {{counter}}
+    </span>
+    <span *ngIf="secondaryIcons!== null && secondaryIcons!== undefined">
+        <span *ngFor="let secondaryIcon of secondaryIcons">
+            <fd-icon *ngIf="!(secondaryIcon.isButton)" class="fd-list__icon" [glyph]="secondaryIcon.icon"
+                [attr.alt]="secondaryIcon.icon" [attr.role]="'img'" (click)="onActionButtonClick($event)">
+            </fd-icon>
+            <button fd-button *ngIf="secondaryIcon.isButton" [attr.role]="'button'"
+                (click)="onActionButtonClick($event)" [compact]="isCompact" [fdType]="'transparent'"
```

Comment:
unnecessary bindings
plz check the whole file

Reply:
updated